src/utils.rs

```diff
@@ -2,7 +2,6 @@ use pbr::ProgressBar;
 use std::error::Error;
 use std::io::Stdout;
 use std::str::FromStr;
-use std::sync::{Arc, Mutex};
 use std::time::Duration;
 
 pub const PROGRESS_REFRESH_RATE_SECS: u64 = 2;
```

Comment:
tiny nit: (I think) As `Duration::from_sec` is a `const fn`, we can do 
```
pub const PROGRESS_BAR_REFRESH_RATE: Duration = Duration::from_secs(2);
``` 
here instead of below.
It looks cleaner to me than how it's currently done, but it's a matter of taste, I guess.

Reply:
Done